docs/quickstart.md

```diff
@@ -2,7 +2,9 @@
 
 The following example sets up the Ark server and client, then backs up and restores a sample application.
 
-For simplicity, the example uses Minio, an S3-compatible storage service that runs locally on your cluster. See [Set up Ark with your cloud provider][3] for how to run on a cloud provider. 
+For simplicity, the example uses Minio, an S3-compatible storage service that runs locally on your cluster. Backups and restores for an on-premises cluster work the same way as the example.
+
+See [Set up Ark with your cloud provider][3] for how to run on a cloud provider. 
```

Comment:
I would change this text and link to say "Set up Ark on your platform" or sthg like that.

Then on the page that it links to, I'd revamp it so it just talks about "Setting up Ark on your platform" -- we can put another link here to the support matrix page.  Then we could have a little section on this page that discusses possible ways to run on-prem (i.e. have a highly-available Minio instance running OUTSIDE of your cluster, be backing up to it, yadda yadda yadda)

Reply:
+1 to "on you platform" wording.